cv32e40x/env/corev-dv/cv32e40x_csr_template.yaml

```diff
@@ -753,10 +936,11 @@
       lsb: 0
 
 ###############################################################################
-# mvendorid, marchid, mimpid and mhartid are temporarily excluded from auto-
-# generation of access testing as all bits in these CSRs are RO, so any attempt
-# to write them causes an illegal instruction exception.  Access modes to these
-# CSRs is tested in a separate, manually written test-program.
+# mvendorid, marchid, mimpid, mhartid, tcontrol and mconfigptr are temporarily
```

Comment:
tcontrol does not belong in this category. It is a MRW CSR. (The same holds for tinfo; both of them are misdocumented in the UM; I will add an issue for that)

Reply:
Thanks and well spotted!
I will update this accordingly